tests/rptest/tests/scaling_up_test.py

```diff
@@ -132,20 +150,72 @@ def create_topics(self, rf, partition_count):
         for spec in topics:
             DefaultClient(self.redpanda).create_topic(spec)
 
-        self.topic = random.choice(topics).name
+        self._topic = random.choice(topics).name
 
         return total_replicas
 
-    @cluster(num_nodes=5)
+    @property
+    def producer_throughput(self):
+        return 100000 if not self.debug_mode else 1000
```

Comment:
100kb/s - seems a bit low, no?

Reply:
i left what was in the test before, good point